UofS-Pulse-Binfo/rawphenotypes
# Rawphenotypes #11   code review

Solved Issue #11.

Please ignore commit item 10 in issue # 11 - Tripal Job Message. This is part of commit Non-code based issue in issue # 11.

To test:
Please clear your cache (Twice if possible) then click each pages in rawphenotypes module.

Rawdata page
Instructions page
Upload page
Backup page
Download page

Look at each page to see that it has no broken image, link etc. and if it has colour scheme plus a warning if module has no project nor data (if module was installed fresh). This is to test that new hook_preprocess that's been replaced with a more specific hook to form/page.

To test revised upload validators in file upload, simply upload a spreadsheet file into backup and upload, while doing so, see terminal to view shortened tripal jobs message.

More notes in each commit.
